src/configs/strings/th.json

```diff
@@ -651,10 +651,6 @@
     "message": "ให้สุขศึกษาในพื้นที่ปฏิบัติงานอย่างน้อย 1 ครั้ง",
     "description": ""
   },
-  "BCC_GOAL_MEASURE": {
-    "message": "จำนวนกิจกรรมการให้สุขศึกษา",
-    "description": ""
-  },
```

Comment:
Can we keep this and ask if the translation needs to be updated? @kahummer

Reply:
ok, cool.